### PR TITLE
Add multiple events and update link

### PR DESCRIPTION
**Pull request by phcarbajal**
added event Holiday NYC 
---

> ✅️ By submitting this PR, I have verified the following

- [x] Checked to [see if a similar PR has already been opened](https://github.com/techqueria/website/pulls) 🤔️
- [x] Reviewed the [contributing guidelines](https://github.com/techqueria/website/blob/master/.github/CONTRIBUTING.md) 🔍️
- [x] Added my name to the bottom of the list under the **Contributors** section in the [README.md](https://github.com/techqueria/website/blob/master/README.md) with a link to my personal website or GitHub profile 👥️ (Optional)

### CODE REVIEW

File: content/english/events/2019-12-12-caminos-into-tech.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Caminos into Tech"
+description: "A conversation with Latinx tech professionals"
+date: 2019-12-12
+image: "/assets/img/events/2019-12-12-honey-panel.jpg"
```
Comment (fvcproductions) [inline content/english/events/2019-12-12-caminos-into-tech.md]: image files should be less than 1MB in size - can you use an online image compressor to reduce the image size - making the width max 1500px helps a lot as well!

right now, the image is almost 5MB which is very big and is not good for the performance of website

File: layouts/partials/content/custom/supporters-showcase.html [hunk 2]
```diff
@@ -132,10 +133,37 @@ <h3 id="past-supporters-and-partners"><span class="mr-sm">🕒</span>Past Suppor
 <!-- Active Patreon Supporters -->
 <div class="content mt-4 u-text--centered">
   <h2 id="patreon-supporters"><span class="mr-sm">🎗️</span>Active Patreon Supporters<span class="ml-sm">🎗️</span></h2>
+</div>
+<!-- Champion Level Supporters -->
```
Comment (fvcproductions) [inline layouts/partials/content/custom/supporters-showcase.html]: cool!

**Comment by fvcproductions** [comment]
let's combine #594 #593 #592 all into this PR if possible

**Comment by phcarbajal** [comment]
What is considered best practice? Multiple branches of small features? Or just pushing into one giant master branch for a PR?

**Comment by phcarbajal** [comment]
compressed the image to be just under 1MB 😢

**Comment by fvcproductions** [comment]
generally speaking, best practice is to break up specific changes into separate PRs so in this case, adding 1 event could be 1 PR, adding another event could be the 2nd PR, etc, etc.

however, you were creating multiple PRs with the same files changed so it wasn't modular anyways

and because you're not changing that many files (less than 15) - it's totally fine to bulk them all in one PR

for transparency, i often just bulk update website with a massive PR which isnt best practice

**Comment by fvcproductions** [comment]
LGTM +1

**Pull request merged**